kai/scm.py

```diff
@@ -58,3 +58,18 @@ def get_branches(self):
 
     def checkout_branch(self, branch_name):
         self.repo.git.checkout(branch_name)
+
+    def diff_exists_for_file(self, old_commit, new_commit, file_path):
```

Comment:
Looks good, if possible it'd be nice to add a short test for this to https://github.com/konveyor-ecosystem/kai/blob/main/tests/test_scm_git_diff.py